Pokedex/Pokedex/Model/Pokemon.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Pokemon.swift
+//  Pokedex
+//
+//  Created by Taylor Lyles on 8/9/19.
+//  Copyright © 2019 Taylor Lyles. All rights reserved.
+//
+
+import Foundation
+
+struct Pokemon: Codable {
+	
+	let name: String
+	let Id: Int
+	let abilities: [Abilities]
+	let type: [Type]
+	let sprites: Sprites
+	
+}
+
+struct Sprites: Codable {
+	let fontDefault: String
+	
+	enum CodingKeys: String, CodingKey {
+		case fontDefault = "font_default"
+	}
+}
+
+struct Type: Codable {
+	let type: String
+}
+
+struct Abilities: Codable {
```

Comment:
take a closer look at the structure of the json for this type (and the others like it)